packages/codegen-ui-react/lib/forms/form-renderer-helper.ts

```diff
@@ -451,7 +492,7 @@ export const addFormAttributes = (
   return attributes;
 };
 
-export const buildOnChangeStatement = (
+export const XXbuildOnChafdafdngeStatement = (
```

Comment:
I presume this is slightly unintentional? haha

Reply:
d'oh! thanks, deleted the whole block